src/Compiler/Service/SemanticClassification.fs

```diff
@@ -209,190 +184,191 @@ module TcResolutionsExtensions =
                         if duplicates.Add m then
                             results.Add(SemanticClassificationItem((m, typ)))
 
-                    resolutions
-                    |> Array.iter (fun cnr ->
```

Comment:
This changed to

```fsharp
sResolutions.CapturedNameResolutions
|> Internal.Utilities.ResizeArray.iter (fun cnr ->
    if range.IsNone || rangeContainsPos range.Value cnr.Range.Start || rangeContainsPos range.Value cnr.Range.End then
```

everything else in the diff below is caused by indendation adjustment.